upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -187,6 +188,22 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		manifests[key] = "addons/" + location
 	}
 
+	if featureflag.EnableExternalDNS.Enabled() {
```

Comment:
You only have a k8s 1.6 manifest, but people can enable this for 1.5. Can we shut it off here for k8s < 1.6 or get a 1.5 manifest?